src/main/java/uk/gov/pay/adminusers/service/ServiceInviteCreator.java

```diff
@@ -90,8 +90,8 @@ public Invite doInvite(InviteServiceRequest inviteServiceRequest) {
 
     private Invite constructInviteAndSendEmail(InviteServiceRequest inviteServiceRequest, InviteEntity inviteEntity, Function<InviteEntity, Void> saveOrUpdate) {
         String inviteUrl = format("%s/%s", linksConfig.getSelfserviceInvitesUrl(), inviteEntity.getCode());
-        inviteEntity.setTelephoneNumber(TelephoneNumberUtility.formatToE164(inviteServiceRequest.getTelephoneNumber()));
-        inviteEntity.setPassword(passwordHasher.hash(inviteServiceRequest.getPassword()));
+        Optional.ofNullable(inviteServiceRequest.getTelephoneNumber()).map(TelephoneNumberUtility::formatToE164).ifPresent(inviteEntity::setTelephoneNumber);
+        Optional.ofNullable(inviteServiceRequest.getPassword()).map(passwordHasher::hash).ifPresent(inviteEntity::setPassword);
```

Comment:
I'm guessing using an `Optional.ofNullable` on the telephone number and password are for backward compatibility at the moment and will be removed once we remove the tel number and password from the UI during account creation?

Reply:
Yes, there will be a period where they may or may not be included in the request.